backend/pkg/portforward/handler_test.go

```diff
@@ -75,14 +74,16 @@ func TestStartPortForward(t *testing.T) {
 		if len(pod.Spec.Containers) > 0 && len(pod.Spec.Containers[0].Ports) > 0 {
 			podName = pod.Name
 			targetPort = fmt.Sprint(pod.Spec.Containers[0].Ports[0].ContainerPort)
+
+			break
 		}
 	}
 
 	require.NotEmpty(t, podName)
 	require.NotEmpty(t, targetPort)
 
 	// start portforward
-	req := &http.Request{}
+	req := httptest.NewRequest(http.MethodPost, "/start-portforward", nil)
```

Comment:
Looks like there's some port forward related changes in here too?

I guess it makes sense to move these portforward related things to a separate commit, and mention them in the PR description too?

Reply:
Aaah, I added that path for testing, forgot to remove it. There is no change made to port forward functionality just some changes in err handling due to `LoadContextsFromFile`, so I think it can be in the same commit. If we move it to separate commit the file will fail due to err handling. Fixing it. Thanks for catching it